### ISSUE TITLE
A way to recognize ignored tests when running them through JUnit/FitNesseRunner

### ISSUE THREAD

**Issue by hansjoachim** [unclebob/fitnesse#710]
Hello

I have been looking a bit at FitNesseRunner and how can run FitNesse tests as part of a build process. While it will report passing and failing tests as expected, I haven't found a way to make it report ignored tests. I have put together a small [proof of concept as an example](https://github.com/hansjoachim/FitNessePOC). I've explained more in-depth in the readme, but at the moment I have one enabled and one ignored tests but it only counts the passing test. If I ignore the second one as well, I only get an error message saying it expected at least one test to run.

So I basically wonder if it is (or would be possible to add) a way to report these tests as ignored to the JUnit results. Currently I would need to run FitNesse and open the wikipage with the test suite in order to tell whether there are any ignored tests. (Or I could run `grep -rin "Prune" FitNesseRoot`, I suppose.)

Reporting the number of ignored tests would be useful so that developers can notice and react to it, it is possible to set threshold levels in continous integration etc.

PS. With regards to the recent changes to CONTRIBUTING.md: this is arguably a question, but to me it looks like a feature request.

**Comment by six42** [comment]
This would be a nice feature.

Just a question: If you don't prune the the second test but comment out the table on the page is it then reported as ignored?

My expierence is that pruned pages like static pages are not passed to the test system and are therfore not visible in the result set. So even running from the Wiki you would not get what you are looking for.

**Comment by hansjoachim** [comment]
>If you don't prune the the second test but comment out the table on the page is it then reported as ignored?

No, I just tested and it reports two tests ran successfully. I'd say this is a slightly different case, because there might be a page which is run as a test but if it doesn't contain any assertions it will never fail. This might be intended, but in cases where it is not I guess it is harder to tell the difference between tests which ran at least one assertion and those which didn't. Though it is potentially useful to be able to track down pages where no assertions are checked.

>My expierence is that pruned pages like static pages are not passed to the test system and are therfore not visible in the result set.

Good point, I didn't consider static pages. Hm... one may not want those registering as ignored tests and create false positives.
